src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
+        <div class="navigation__logo">
+          <img
+            src="images/logo.png"
+            alt="moyo logo"
+            srcset=""
```

Comment:
The 'srcset' attribute is empty. If it's not needed, consider removing it. Otherwise, provide appropriate values.